discovery-provider/scripts/start.sh

```diff
@@ -72,6 +73,20 @@ audius_discprov_loglevel=${audius_discprov_loglevel:-info}
 # used to remove data that may have been persisted via a k8s emptyDir
 export audius_prometheus_container=server
 
+# run alembic migrations
```

Comment:
We tried moving migrations here before and that caused a bunch of problems because not all service providers were using the `start.sh` thing yet. I think that will all be fixed in the new compose setup, but wanted to flag & cc @cheran-senthil

Reply:
Yeah I think I experienced some problems in dev where flask server failed to boot because of race condition behavior: celery worker needed to create a row for flask to boot... but since flask and celery were booting at same time it was more likely to happen... there was some discussion with @dmanjunath and @rickyrombo in the slack about this.

I believe the main reason I moved it out to `start.sh` is to ensure migrations ran before indexer started... as indexer may depend on columns created in migration.  Granted indexer crash behavior should hopefully make it so that it crashes until migrations finish and then works... but on some level it'd be nice to have predictable behavior around migrations and the code that depends on them.

Anyway some ideas:
* add a sleep to `dev-server.sh` to simulate migration time so flask boots okay
* undo this change and let es-indexer crash if there's any migration dependency
* undo this change and move es-indexer boot into respective shell scripts after migrations.

Reply:
yeah as far as the worker is concerned, unfortunately i don't think we have a way to guarantee it starts up after the migrations finish since the [k8s manifests start celery worker separately](https://github.com/AudiusProject/audius-k8s-manifests/blob/master/audius/discovery-provider/discovery-provider-deploy.yaml#L66). the audius-docker-compose work changes that so everything is kicked off via start.sh. 

but for now to avoid coupling these changes, my take would be to move the es-indexer start to the shell scripts after the migration runs. I actually don't remember exactly what broke last time, these look functionally identical but that's definitely the safest option for now